records_mover/db/bigquery/load_job_config_options.py

```diff
@@ -121,7 +124,8 @@ def load_job_config(unhandled_hints: Set[str],
 
 
 def add_load_job_csv_config(unhandled_hints: Set[str],
-                            hints: RecordsHints,
+                            # TODO: Use validated hints
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker